services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/rest/_health.py

```diff
@@ -7,7 +7,7 @@
 from models_library.errors import RABBITMQ_CLIENT_UNHEALTHY_MSG
 from servicelib.rabbitmq import RabbitMQClient
 
-from ..modules.rabbitmq import get_rabbitmq_client_from_request
+from ...modules.rabbitmq import get_rabbitmq_client_from_request
```

Comment:
just a thought but, at some point here you should also check that access to redis/postgres works.

Reply:
Yes valid point, mainly postgres I think Redis is not a crucial application for RUT. I will add a note in the code for now.